### PR TITLE
update default data to r/aww

### PR DESCRIPTION
**Pull request by dsteedRTI**
Update the default data to sample data from [r/awww](https://www.reddit.com/r/aww/) and link to the proper data when setting up the database.

In `envs/dev` run `./rebuild.sh` to rebuild the database with new data.

### CODE REVIEW

**Comment by AstridKery** [comment]
Getting one new test failure in `test/core_utils/test_core_utils_model.py` (the two in `test/api/test_api.py` are from the maintenance branch)
```
_________________________________ test_create_tfidf_matrix _________________________________

test_tfidf_matrix = {'0': [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, ...], '1': [0.0, 0.0, 0.0, 0.0, 0.7143086710658769, 0.0, ...], '10': [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ...], '100': [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ...], ...}

    def test_create_tfidf_matrix(test_tfidf_matrix):
        # UPDATE: is now saved as a dictionary of lists
        assert isinstance(test_tfidf_matrix, dict)
>       assert len(test_tfidf_matrix) == 285
E       AssertionError: assert 982 == 285
E        +  where 982 = len({'0': [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, ...], '1': [0.0, 0.0, 0.0, 0.0, 0.7143086710658769, 0.0, ...], '10': [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ...], '100': [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ...], ...})

test/core_utils/test_core_utils_model.py:21: AssertionError
```

**Comment by AstridKery** [comment]
@dsteedRTI I found one issue with the tests (above) but otherwise looks good!

**Comment by dsteedRTI** [comment]
I'm not 100% sure on this, but I think that test assertion might need to be changed. I think it's testing if the length equals the amount of training data. The old files had 285 entires, but the r/awww files have 982. Does this sound right to you?

File: backend/django/core/data/SemEval-2016-Task6/test-awww.csv [hunk 1]
```diff
@@ -0,0 +1,72 @@
+Text,Label
```
Comment (AstridKery) [inline backend/django/core/data/SemEval-2016-Task6/test-awww.csv:1]: One thing I didn't notice earlier, the test-aww.csv file should go in a new folder `backend/django/core/data/reddit_sample_awww`. The semeval folder is specifically for data to do with the SemEval data challenge which is the source of the feminism tweet data.

Comment (dsteedRTI) [reply backend/django/core/data/SemEval-2016-Task6/test-awww.csv]: done

File: backend/django/core/management/commands/seed_SMART.py [hunk 2]
```diff
@@ -100,15 +100,15 @@ def handle(self, *args, **options):
...
             multi_user_project = seed_project(
                 creator=root,
                 name='Three User Project',
                 description="This is a project that is coded by three different users.  No labels were in the data file to begin with.",
                 data_file='./core/data/test_files/test_no_labels.csv',
-                label_list=['Good', 'Bad'],
+                label_list=['about cats', 'not about cats', 'unclear'],
                 perm_list=[user1, test_user],
                 classifier='logistic regression')
             seed_project(
```
Comment (AstridKery) [inline backend/django/core/management/commands/seed_SMART.py:114]: This last project needs to have the new set of labels too

Comment (dsteedRTI) [reply backend/django/core/management/commands/seed_SMART.py:125]: done

**Pull request merged**